models/__init__.py

```diff
@@ -0,0 +1,9 @@
+from .bioreplicate import Bioreplicate
+from .excel_file import ExcelFile
+from .experiment import Experiment
+from .measurement import Measurement
+from .metabolite import Metabolite
+from .project import Project
+from .study import Study
+from .submission import Submission
+from .taxon import Taxon
```

Comment:
This file was created as a proxy for model imports. Because the models refer to each other, it's important that all of them are loaded before one of them is used. For instance, calling `measurement.bioreplicate` is going to need the `Bioreplicate` class to be evaluated even if it's not explicitly imported.

Models should now be imported as `from models import <ClassName>`. This needs to eventually be documented extensively with explanations and links to the documentation.